app/helpers/registrations_helper.rb

```diff
@@ -0,0 +1,11 @@
+module RegistrationsHelper
+  def sortable(title, column, direction)
```

Comment:
It's a global namescape, perhaps volunteers_sortable? Also, perhaps volunteers_sortable_select or whatever HTML element it is.